tools/bigquery-zos-mainframe-connector/USING_WITH_GCS.md

```diff
@@ -0,0 +1,89 @@
+# Using with Cloud Storage
+
```

Comment:
also recommend a brief description of the functionality here and maybe some of the history.
"This project was created for loading Bigquery - the primary use case is copying a mainframe dataset directly to GCS, then invoking the associated gRPC service to transcode the mainframe dataset to ORC"
Users should pay attention to the amount of retention they'd like to have on the original mainframe datasets.
I think they're deleted after being successfully transcoded - please confirm that and note it here.